editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -4080,6 +4080,12 @@ void VisualShaderEditor::_show_members_dialog(bool at_mouse_pos, VisualShaderNod
 		saved_node_pos_dirty = false;
 		members_dialog->set_position(graph->get_screen_position() + Point2(5 * EDSCALE, 65 * EDSCALE));
 	}
+
+	if (members_dialog->is_visible()) {
+		members_dialog->grab_focus();
+		return;
+	}
+
```

Comment:
Since this dialog is now exclusive, I guess this situation should no longer happen?

May not hurt to keep it for safety.

Reply:
Yes this should be removed, if there is any related bug in the future I will fix it.